packages/manager/modules/pci/src/projects/project/sidebar/translations/Messages_fr_FR.json

```diff
@@ -26,5 +26,5 @@
   "cloud_sidebar_project_management_billing_control": "Facturation",
   "cloud_sidebar_project_management_credit_vouchers": "Crédit & Vouchers",
   "cloud_sidebar_project_management_contact_rights": "Contacts & Droits",
-  "cloud_sidebar_project_management_settings": "Project settings"
+  "cloud_sidebar_project_management_settings": "Paramètres du projet"
```

Comment:
The sidebar label should be in EN (we should be consistent about this, cf: https://github.com/ovh-ux/manager/pull/430/files#diff-949befe93701e0e9edb1237b0e9c4af4R10)

Reply:
/cc @fchabrolle